composer.json

```diff
@@ -21,11 +21,11 @@
     ],
     "require": {
         "php" : "^7.0 || ^8.0",
-        "psr/http-message": "^1.0"
+        "psr/http-message": "^1.0 || ^2.0"
     },
     "require-dev": {
         "scrutinizer/ocular": "~1.1",
-        "guzzlehttp/psr7": "^1.3",
+        "guzzlehttp/psr7": "^1.3 || ^2.0",
```

Comment:
I'm fine with dropping support for guzzle ^1.3 and http-message ^1.0, what do you think? :)

Then we don't have to mark the test as skipped?

Reply:
Yes, I also think it's no problem.

In that case, `psr/http-message:^2.0` requires `php:^7.2`, so I also removed it from the CI php version. Instead, I added a version newer than 8.0 to CI. You can see results [here](https://github.com/KentarouTakeda/hansott-psr7-cookies/actions/runs/7441738566).

https://packagist.org/packages/hansott/psr7-cookies/php-stats

Right now, the dropped versions seems to be used by nobody :)